### DISCUSSION TITLE
Roadmap project board

### DISCUSSION THREAD

**Discussion by DavidMStraub** [gramps-project/gramps-web#550] [General]
Hi @Nick-Hall, I created https://github.com/orgs/gramps-project/projects/1 as a unified roadmap for Gramps Web to give some more visibility to users what is being worked on. But I just realized the board is not publicly visible and I cannot change it. Can you please change the visibility to "Public" here https://github.com/orgs/gramps-project/projects/1/settings? Thanks!

**Comment by Nick-Hall** [comment]
OK.  It should be public now.  Let me know if any other settings need changing.
